static/src/js/utils/convertFormDataToRdf.js

```diff
@@ -0,0 +1,118 @@
+import { XMLBuilder } from 'fast-xml-parser'
+import removeEmpty from '@/js/utils/removeEmpty'
+
+/**
+ * Converts form data to RDF XML format
+ * @param {Object} formData - The form data to be converted
+ * @returns {string} The RDF XML string
+ */
+export const convertFormDataToRdf = (formData, userNote, scheme, uid) => {
+  // Initialize XML builder with specific options
+  const builder = new XMLBuilder({
+    ignoreAttributes: false,
+    format: true,
+    suppressEmptyNode: true,
+    attributeNamePrefix: '@_'
+  })
+  /**
+   * Ensures that the input is an array
+   * @param {*} data - The input data
+   * @returns {Array} An array of the input data
+   */
+  const ensureArray = (data) => {
+    if (Array.isArray(data)) {
+      return data
+    }
+
+    if (data) {
+      return [data]
+    }
+
+    return []
+  }
+
+  const createNewChangeNote = () => {
+    const currentDate = new Date().toISOString().split('T')[0]
+
+    return `Date=${currentDate}\nUser Id=${uid}\nUser Note=${userNote}`
```

Comment:
Use space instead of \n to separate, that is how the other changes notes look.

Reply:
Done